{
    "error": {
        "error_code": 5,
        "error_msg": "User authorization failed: invalid access_token (4).",
        "request_params": [
            {
                "key": "user_ids",
                "value": "534672230,244960181,548444447"
            },
            {
                "key": "random_id",
                "value": "33323"
            },
            {
                "key": "v",
                "value": "5.92"
            },
            {
                "key": "method",
                "value": "messages.send"
            },
            {
                "key": "oauth",
                "value": "1"
            }
        ]
    }
}